doc/fr/installation/prerequisites.rst

```diff
@@ -74,24 +74,29 @@ Sélectionner votre architecture
-+----------------------+-------------------------+---------------------------+----------------+---------------+
-|     20000 - 50000    |       2000 - 5000       | 1 central + 2 collecteurs |  4 vCPU / 8 GB | 4 vCPU / 2 GB |
-+----------------------+-------------------------+---------------------------+----------------+---------------+
-|     50000 - 100000   |       5000 - 10000      | 1 central + 3 collecteurs |  4 vCPU / 8 GB | 4 vCPU / 2 GB |
-+----------------------+-------------------------+---------------------------+----------------+---------------+
++----------------------+-------------------------+----------------------------+----------------+---------------+
+|  Nombre de services  |  Nombre d'hôtes estimé  |  Nombre de collecteurs     |  Central       |  Collecteur   |
++======================+=========================+============================+================+===============+
+|           < 500      |           50            |        1 central           |  1 vCPU / 1 GB |               |
++----------------------+-------------------------+----------------------------+----------------+---------------+
+|       500 - 2000     |         50 - 200        |        1 central           |  2 vCPU / 2 GB |               |
++----------------------+-------------------------+----------------------------+----------------+---------------+
+|      2000 - 7000     |        200 - 700        | 1 central + 1 collecteur   |  4 vCPU / 4 GB | 2 vCPU / 4 GB |
++----------------------+-------------------------+----------------------------+----------------+---------------+
+|      7000 - 14000    |        700 - 1400       | 1 central + 2 collecteurs  |  4 vCPU / 8 GB | 2 vCPU / 4 GB |
++----------------------+-------------------------+----------------------------+----------------+---------------+
+|     14000 - 21000    |       1400 - 2100       | 1 central + 3 collecteurs  |  4 vCPU / 8 GB | 4 vCPU / 4 GB |
++----------------------+-------------------------+----------------------------+----------------+---------------+
+|     21000 - 28000    |       2100 - 2800       | 1 central + 4 collecteurs  |  4 vCPU / 8 GB | 4 vCPU / 4 GB |
++----------------------+-------------------------+----------------------------+----------------+---------------+
+|          ...         |           ...           |             ...            |      ...       |      ...      |
++----------------------+-------------------------+----------------------------+----------------+---------------+
+|     84000 - 91000    |       8400 - 9100       | 1 central + 13 collecteurs |  4 vCPU / 8 GB | 4 vCPU / 4 GB |
++----------------------+-------------------------+----------------------------+----------------+---------------+
 
 .. note::
-    Les vCPU doivent avoir une fréquence avoisinant les 3 GHz
+    Un collecteur peut superviser en moyenne 7000 services actifs.
+    Les vCPU doivent avoir une fréquence avoisinant les 3 GHz.
 
```

Comment:
add sentence saying that the number of CPU globally depends on the complexity of the checks. (if you use a connector, or perform a lot of system/third-party binary call) you may want to add more CPUs.